manifest.json

```diff
@@ -415,6 +415,13 @@
         "url": "https://github.com/yui/yuicompressor/releases/download/v2.4.8",
         "files": ["yuicompressor-2.4.8.jar"],
         "targetDirPrefix": "yuicompressor-"
+      },
+      "browsermob-proxy": {
```

Comment:
Can you note which is which? Or rather, in the install_third_party script can you specify whether that is the binary or the python API?